src/frontend/Komponenter/Journalføring/JournalføringApp.tsx

```diff
@@ -162,20 +157,11 @@ const JournalføringAppContent: React.FC<JournalføringAppProps> = ({
         // eslint-disable-next-line
     }, [fagsak]);
 
-    const skalBeOmBekreftelse = (
-        erNyBehandling: boolean,
-        harStrukturertSøknad: boolean,
-        ustrukturertDokumentasjonType: UstrukturertDokumentasjonType | undefined
-    ) => {
-        if (harStrukturertSøknad) {
-            return !erNyBehandling;
-        } else if (ustrukturertDokumentasjonType === UstrukturertDokumentasjonType.PAPIRSØKNAD) {
-            return !erNyBehandling;
-        } else if (ustrukturertDokumentasjonType === UstrukturertDokumentasjonType.ETTERSENDING) {
-            return erNyBehandling;
+    const skalBeOmBekreftelse = (erNyBehandling: boolean) => {
+        if (!erNyBehandling) {
+            return journalResponse.harStrukturertSøknad || erPapirsøknad;
```

Comment:
Vil bare be saksbehandler bekrefte hvis man journalfører en søknad på en eksisterende behandling